otx/cli/tools/deploy.py

```diff
@@ -82,11 +82,10 @@ def main():
 
     deployed_model = ModelEntity(None, environment.get_model_configuration())
 
-    if "save_model_to" not in args or not args.save_model_to:
-        args.save_model_to = str(config_manager.workspace_root / "model-deployed")
-    os.makedirs(args.save_model_to, exist_ok=True)
+    output_path = Path(args.output) if args.output else config_manager.output_path
```

Comment:
how about using `config_manager.output_path`? Same algorithm exists there.

Reply:
I fixed this. https://github.com/openvinotoolkit/training_extensions/pull/1946/commits/2cb8133579f0f6a36f78f4a32ef2a5287d1008e8